### ISSUE TITLE
Thumbnail preview won't work. I've read and did what worked for others earlier and ueberzug is working a ok

### ISSUE THREAD

**Issue by onlyvip** [pystardust/ytfzf#95]
I edited line 254 to
preview  window "left"
After which I now have tab view but the thumbnail preview won't work.
Here's some extra info 
I have ueberzug running I'm using it with ranger. Also tried it with the ueberzug script and it works. 
Any help is appreciated.

I am on bspwm. MX linux debian buster 10. I have no swallow like behaviour.
Tried in urxvt,termite,roxterm and xfce4-terminal

**Comment by paniash** [comment]
Hi! Are you using the fish shell?

**Comment by paniash** [comment]
I'd recommend installing ueberzug from `pip` instead since the debian repos might be outdated.

**Comment by onlyvip** [comment]
No I'm on bash.

I have ueberzug 1.18.6
I have this script working perfectly with 

```
#!/bin/sh
   2   │ 
   3   │ [ -z "$1" ] && { echo "Usage: ubtest.sh  <image>" ; exit; }
   4   │ 
   5   │ FIFO="ytfzf-test-pipe"
   6   │ 
   7   │ [ -e "$FIFO" ] || mkfifo "$FIFO"
   8   │ 
   9   │ ueberzug layer --parser json <> "$FIFO" &
  10   │ 
  11   │ read ueberzug_info << EOF
  12   │ { "action": "add", "identifier": "test", "path": "$1", "x": 0, "y": 0, "scaler": "fit_contain", "width": 50, "height": 50 }
  13   │ EOF
  14   │ 
  15   │ echo "$ueberzug_info" > "$FIFO"
  16   │ sleep 5
  17   │ 
  18   │ rm "$FIFO"

```


And it outputs an image perfectly.
I have ranger working perfectly with ueberzug. So there is something else not working. 
I have edited line 259 in ytfzf
To this
```
--preview-window "left"
```

**Comment by paniash** [comment]
@onlyvip Like I said, ueberzug 18.1.9 is the latest version and I'd recommend installing using `pip`.

**Comment by onlyvip** [comment]
> @onlyvip Like I said, ueberzug 18.1.9 is the latest version and I'd recommend installing using `pip`.

I have ueberzug 1.18.9 now. Still no previews.

**Comment by Euro20179** [comment]
if you have other processes using ueberzug, try ending all processes using ueberzug and run ytfzf alone

**Comment by onlyvip** [comment]
> if you have other processes using ueberzug, try ending all processes using ueberzug and run ytfzf alone

There are none running in the background. I tried ytfzf after a fresh login that's why I can tell

**Comment by onlyvip** [comment]
![image](https://i.imgur.com/PDwmuWh.png)
Here's a error when I try to run 
```
ytfzf -t 
```

**Comment by Euro20179** [comment]
what version of mpv are you using

**Comment by onlyvip** [comment]
> what version of mpv are you using

https://termbin.com/ph3r

**Comment by Euro20179** [comment]
try updating mpv, i'm on version 0.32.0

**Comment by onlyvip** [comment]
> try updating mpv, i'm on version 0.32.0

F.
ok

**Comment by onlyvip** [comment]
> try updating mpv, i'm on version 0.32.0

now i have updated mpv to  STDIN
───────┼──────────────────────────────────────────────────────────────────────────────
   1   │ mpv 0.32.0 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
   2   │  built on UNKNOWN
   3   │ ffmpeg library versions:
   4   │    libavutil       56.22.100
   5   │    libavcodec      58.35.100
   6   │    libavformat     58.20.100
   7   │    libswscale      5.3.100
   8   │    libavfilter     7.40.101
   9   │    libswresample   3.3.100
  10   │ ffmpeg version: 4.1.6-1~mx19+2
  11   │ 
───────┴──────────────────────────────────────────────────────────────────────────────

still no previews🤔

**Comment by Euro20179** [comment]
if you're on debian try doing `sudo apt update && sudo apt upgrade`, some of your packages seem out of date, other than this idk

**Comment by onlyvip** [comment]
> if you're on debian try doing `sudo apt update && sudo apt upgrade`, some of your packages seem out of date, other than this idk

everything is as updated as it can be

**Comment by onlyvip** [comment]
> if you're on debian try doing `sudo apt update && sudo apt upgrade`, some of your packages seem out of date, other than this idk

the mpv version i had is what comes in standard repo of debian. the version you had is not stable one. but as i am on mx we have test repo where mpv verrsion 0.32 was available. so i have updated versions according to the debian buster model. 
and have also updated ueberzug to the latest by getting it from pip and updated mpv by updating it from mx test repo. 
imo the version thing isnt a problem. something else is causing it.

**Comment by Euro20179** [comment]
> imo the version thing isnt a problem. something else is causing it.

Probably, and I'm out of ideas for now

**Comment by onlyvip** [comment]
![image](https://i.imgur.com/b7PDyRi.png)
i tried it normally and this error says i dont have any scripts of mpv would that cause this preview problem?

**Comment by westofer** [comment]
Do you get not previews or does mpv not work, or both?

**Comment by onlyvip** [comment]
> Do you get not previews or does mpv not work, or both?

as mentioned in the title. just the previews are not working everything else is working. mpv starts. video plays well. the tabbed view shows up just the previews arent working
like this ![image](https://i.imgur.com/XhDTGmz.png)

**Comment by onlyvip** [comment]
could it be due to the change in  this ?


video_menu_img () {
	title_len=400
	#for thumbnails, fzf is always used
	format_video_data "$*" | fzf -m --tabstop=1 --bind change:top --delimiter="$(printf "\t")" --nth=1,2 $FZF_DEFAULT_OPTS \
		--layout=reverse --preview "sh $0 -U {}" \
                
--       	--preview-window "$PREVIEW_SIDE:50%:noborder:wrap"
++           --preview-window "$PREVIEW_SIDE"
                ^
}

**Comment by Euro20179** [comment]
> if it is from mpv can you test with --no-config option?

I think to use mpv with --no-config, run this command

`YTFZF_PLAYER="mpv --no-config" ytfzf <search>`

> could it be due to the change in this diff?

no probably not

**Comment by Euro20179** [comment]
I have a feeling the image is being translated off the screen somehow, because in testing this happened to me, we slightly changed how images are shown in the latest version, although I don't think it should be different, but if you're running the tag release, or an older version you can try the latest commit, and if you're running the latest commit, try [this version](https://github.com/pystardust/ytfzf/tree/0223de84989718d4f8b737afa172b35091cc8bc9)

**Comment by onlyvip** [comment]
> I have a feeling the image is being translated off the screen somehow, because in testing this happened to me, we slightly changed how images are shown in the latest version, although I don't think it should be different, but if you're running the tag release, or an older version you can try the latest commit, and if you're running the latest commit, try [this version](https://github.com/pystardust/ytfzf/tree/0223de84989718d4f8b737afa172b35091cc8bc9)

so is this correct,
```
git clone https://github.com/pystardust/ytfzf.git
```
?

**Comment by Euro20179** [comment]
i'm not sure but i know this works, 
`wget https://raw.githubusercontent.com/pystardust/ytfzf/0223de84989718d4f8b737afa172b35091cc8bc9/ytfzf`
`chmod +x ytfzf`

**Comment by onlyvip** [comment]
>  `chmod +x ytfzf`

i ran this script and i got this error.
![image](https://i.imgur.com/GyZSbhm.png)

after which i edited the script from this,
![image3](https://i.imgur.com/QdX0GOM.png)
to this
![image2](https://i.imgur.com/4I8E89E.png)

still no previews.
![chloesearch](https://i.imgur.com/jdvlO1h.png)
p.s. i have the latest youtube-dl(frrom pip), ueberzug(from pip) and mpv(version 32 from mx test repo).

**Comment by Euro20179** [comment]
although this probably has nothing to do with thumbnails, what version of fzf do you have

there was a similar [issue](https://github.com/pystardust/ytfzf/issues/77) and i suggested these 2 solutions, deleting .Xauthority seemed to work
* deleting .Xauthority and relogging
* `DISPLAY=:0 ytfzf search`

**Comment by onlyvip** [comment]
> although this probably has nothing to do with thumbnails, what version of fzf do you have
> 
> there was a similar [issue](https://github.com/pystardust/ytfzf/issues/77) and i suggested these 2 solutions, deleting .Xauthority seemed to work
> 
>     * deleting .Xauthority and relogging
> 
>     * `DISPLAY=:0 ytfzf search`

i am using this version 
https://github.com/pystardust/ytfzf/issues/95#issuecomment-793288442

**Comment by Euro20179** [comment]
no, what version of [fzf](https://github.com/junegunn/fzf) do you have

**Comment by onlyvip** [comment]
> no, what version of [fzf](https://github.com/junegunn/fzf) do you have

0.17.5

now you will reply with me to update it correct? leme update it then 😂
btw after deleting xathority and logging back in i ran it like you told earlier
![image](https://i.imgur.com/gcyqwuV.png)
and no previews still.
leme try with new fzf now.
so now i deleted fzf 17.5. got 0.25.1.
and 
now i got previews!😇
![chloebaby](https://i.imgur.com/hmZigek.png)

**Comment by onlyvip** [comment]
thanks guys. i am gratefull to you all!✌

**Comment by Euro20179** [comment]
yw, and honestly i'm a bit surprised it was because of the version of fzf
